### PR TITLE
Zanzana: Initial schema loading

### PR DESCRIPTION
**Pull request by alexanderzobnin**
**What is this feature?**
Initialize store and basic schema. This is done during client construction.

We also try to find a fitting authorization model, this part probably needs to be improved.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/identity-access-team/issues/739

**Special notes for your reviewer:**

TODO:
- [ ] Generate schema from resource definition
- [ ] Refactor schema loading
- [ ] Inject `AuthorizationModelID` and `storeUID` into client calls (pass during client init probably)

### CODE REVIEW

File: pkg/services/authz/zanzana/client/client.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+		o(c)
+	}
+
+	if c.logger == nil {
+		c.logger = log.NewNopLogger()
+	}
+
+	if c.tenantID == "" {
+		// TODO what should be the default value?
+		c.tenantID = "default"
```
Comment (Jguer) [inline pkg/services/authz/zanzana/client/client.go:54]: Are we namespacing as in stackid:23 for when it's used?

If so I guess we could do stackid:default

**Review by Jguer — APPROVED** [review]

File: pkg/services/authz/zanzana/client/client.go (hunk 1 above)
Comment (kalleep) [reply pkg/services/authz/zanzana/client/client.go:54]: Yeah this is a tricky one. I think for stacks it probably should match namespaces, e.g. `stack-<id>`.

**Comment by eleijonmarck** [comment]
@Jguer @kalleep looking at these schemas. Should we include a way to run the playground for openfga?

I think the best would be to have README.md with the instruction for running and setting up the schema in the openfga playground

**Comment by kalleep** [comment]
> @Jguer @kalleep looking at these schemas.
> 
> Should we include a way to run the playground for openfga?
> 
> I think the best would be to have README.md with the instruction for running and setting up the schema in the openfga playground

No we have not but that is probably something we could do

**Pull request merged**